.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+!!! Please test the code with https://github.com/babylonlabs-io/finality-provider
+
+Steps to test:
+1. `cd finality-provider`
+2. `git checkout base/consumer-chain-support`
+3. modify the `go.mod` to use the tip commit hash of the branch for this PR
+4. `go mod tidy`
+5. run `make test-e2e-op`
```

Comment:
Can these be enforced in CI instead?

Reply:
good point. we can do that. we just need to write a script to automate this step:

```
3. modify the `go.mod` to use the tip commit hash of the branch for this PR
```

Reply:
we actually have a task https://github.com/babylonlabs-io/finality-gadget/issues/2

Reply:
Hmm ok. Why not using this branch in `go.mod` directly?

Reply:
By `go.mod` which repository are you referring to?

Reply:
`finality-provider`. the 2-4 steps below happen in the FP dir but it's triggered by the CI in this repo:

1. `cd finality-provider`
2. `git checkout base/consumer-chain-support`
3. modify the `go.mod` to use the tip commit hash of the branch for this PR
4. `go mod tidy`
5. run `make test-e2e-op`

Reply:
Hmm I think now I get it. all good then 👍